# motion def:
motion_file: deterministic_error_calib.csv
# IMU params:
imu: 1
# sample frequency of simulated GNSS/IMU data:
sample_frequency:
    imu: 100.0
    gps: 10.0
# topic name:
topic_name: /sim/sensor/imu
# output rosbag path:
output_path: /home/lxq/dataset/catkin_multi/shnelan-multi-sensor-fusion/Sensor_multi_fusion_DEBUG/workspace-ch6/src/9_data/deterministic_error
# output name:
output_name: data.csv